fix: interactive styles on secondary button inside a ButtonGroup (#1938)
sumup-oss/circuit-ui: .changeset/shiny-otters-smash.md

---

> '@sumup/circuit-ui': minor

Reviewer: According to [semantic versioning](https://semver.org/), a backward-compatible bug fix should be a "patch":

```suggestion
'@sumup/circuit-ui': patch
```

---

> fix hover styles on secondary button inside a button group

Reviewer: ```suggestion
Fixed the interactive styles of the secondary button inside a `ButtonGroup`.
```
